vars/acs74.yml

```diff
@@ -16,3 +16,17 @@ supported_os:
   Ubuntu:
     versions:
       - 22.04
+
+acc_version: 8.0.0
+acc_artifact_id: alfresco-control-center
+acc_repository: "{{ nexus_repository.releases }}"
+acc_url: >-
+  {{ acc_repository }}/{{ acc_artifact_id }}/{{ acc_version }}/{{ acc_artifact_id }}-{{ acc_version }}.zip
+acc_checksum: sha1:{{ acc_url }}.sha1
```

Comment:
to avoid duplication lets have a vars/acs-post.yml (whatever) with those and include_var it after loading the first one?

Reply:
moved to playbook group var instead